src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -512,6 +516,28 @@ To rename the `_astro` path, specify a new directory in `build.assets`.
 }
 ```
 
+<p>
+**If use `Record<string, string>`**
+</p>
+
+Applicable to multiple CDN scenarios. If you static file use different CDN, you can configure it like the example below.
+
+For example, if static file is js file, assets prefix use `https://js.cdn.example.com`. If static file is css file, assets prefix use `https://css.cdn.example.com`. Other assets prefix use `https://cdn.example.com`.
+
+```js
+{
+  build: {
+    assetsPrefix: {
+      'js': 'https://js.cdn.example.com',
+      'css': 'https://js.cdn.example.com',
+      'defaultAssetsPrefix': 'https://cdn.example.com'
+    }
+  }
+}
+```
+
+Object key is file extension, value is CDN assets prefix path. `defaultAssetsPrefix` is default assets prefix cdn value set. You can set the default values of other files by setting the value of `defaultAssetsPrefix`. If `defaultAssetsPrefix` is not set, when the file extension is not found in the `assetsPrefix` object, it will be set to the empty string `''` by default.
```

Comment:
```suggestion
The object key is the file extension, and its value is the CDN prefix path. You can set the default values of all other files by setting the value of `defaultAssetsPrefix`. If `defaultAssetsPrefix` is not set, when the file extension is not found in the `assetsPrefix` object, it will be set to the empty string `''` by default.
```
A little bit of rewording